examples/webgl_materials_envmaps_parallax.html

```diff
@@ -356,7 +356,8 @@
 
 				boxProjectedMat.onBeforeCompile = function ( shader ) {
 
-					shader.uniforms.cubeMapSize = { value: new THREE.Vector3( 200, 100, 100 ) };
+					//these parameters are wrt the cubeCamera texture
```

Comment:
What's wrt?

Reply:
with respect to*

Reply:
I see. Maybe better using plain English?

```js
// these parameters are for the cubeCamera texture
```